### PR TITLE
[WIP][Maintenance] Update AP to 2.7@rc

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

This PR is meant to spot issues with upcoming release of Api Platform

### CODE REVIEW

**Comment by lchrusciel** [comment]
@soyuka we are still fighting with our builds, so I can't deduct anything yet :/

**Comment by lchrusciel** [comment]
@soyuka I'm checking my hypothesis, but probably AP2.7 is green for us despite static analysers

File: src/Sylius/Bundle/ApiBundle/Serializer/ShippingMethodNormalizer.php [hunk 1]
```diff
@@ -55,7 +55,9 @@ public function normalize($object, $format = null, array $context = [])
 
         $filters = $request->attributes->get('_api_filters');
         if (null === $filters) {
+            /** @psalm-suppress InternalMethod **/
             $queryString = RequestParser::getQueryString($request);
+            /** @psalm-suppress InternalMethod **/
             $filters = $queryString ? RequestParser::parseRequestParams($queryString) : null;
```
Comment (coldic3) [inline src/Sylius/Bundle/ApiBundle/Serializer/ShippingMethodNormalizer.php:61]: It looks like something that could be replaced with:
```
parse_str($request->server->get('QUERY_STRING', ''), $filters);
```

**Review by coldic3 — APPROVED** [review]

File: composer.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
         "ext-hash": "*",
         "ext-intl": "*",
         "ext-json": "*",
-        "api-platform/core": "^2.6",
+        "api-platform/core": "^2.7@rc",
```
Comment (vasilvestre) [inline composer.json:32]: We can use the non-rc version now that it's been released

**Comment by GSadee** [comment]
Closing in favour of #14386

**Pull request closed without merging**